### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md [hunk 1]
```diff
@@ -8,29 +8,31 @@ weight: 0
...
+Managing a second Tyk Gateway with our [tyk-pro-docker-demo](https://github.com/TykTechnologies/tyk-pro-docker-demo) is a case of mounting the `tyk.conf` file into a new volume and declaring a new Gateway service but exposed on a different port.
+You will need to make some minor modifications to `docker-compose.yml` and `docker-local.yml` and start your services as usual with `docker-compose -f docker-compose.yml -f docker-local.yml up`.
 
 
 > **Please note**: This will only work with an appropriate license. The free license is for development purposes and would allow running Tyk's licensed platform with only one Gateway. If you want to test Tyk with more please contact us by this email   <info@tyk.io> and we will be happy to discuss your case and PoC requirements as well as providing a short period license.
 
 
 
-1. Add the following to docker-local.yml
-    ```yml
```
Comment (spicydarragh) [inline tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md]: Do we need the yml here?

File: tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md [hunk 2]
```diff
@@ -8,29 +8,31 @@ weight: 0
 ---
 
 
-Managing a second Tyk Gateway with our [tyk-pro-docker-demo](https://github.com/TykTechnologies/tyk-pro-docker-demo) is a case of mounting the config file into a new volume and declaring a new gateway service but exposed it on a different port.
```
Comment (spicydarragh) [inline tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md]: exposing

File: tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/frequently-asked-questions/two-gateways-with-docker-compose.md]: Missed that. Will sort

**Review by spicydarragh — APPROVED** [review]
Thanks Mark!

**Pull request merged**